commands/ci/index.js

```diff
@@ -26,7 +26,11 @@ const cmd = {
       char: 'w',
       hasValue: false,
       description: 'keep running and watch for new and update tests'
-
+    },
+    {
+      name: 'json',
```

Comment:
Sure, I can add that. I was using [the other `--json` output in this plugin](https://github.com/heroku/heroku-ci/blob/fe94a0a736ecb71efdaac5b7543bd3b8ec1d4766/commands/ci/config-index.js#L38-L41) as a guide, so maybe that needs an update too?